### ISSUE TITLE
DataProofVerifier isn't moved into relay testing folder during build

### ISSUE THREAD

**Issue by NOOMA-42** [social-tw/social-tw-website#51]
**Describe the bug**
Artifact for contract "DataProofVerifier" not found. This might impact CICD in branch test/login-relay, test/post-relay

**To Reproduce**
clone fresh repo and build

**Expected behavior**

**Screenshots**
```
 1) LOGIN /login
       "before each" hook for "/api/user should catch redirection":
     HardhatError: HH700: Artifact for contract "DataProofVerifier" not found. 
      at Artifacts._handleWrongArtifactForContractName (/Users/paulyu/opensource_project/social-tw-website/node_modules/hardhat/src/internal/artifacts.ts:702:11)
      at Artifacts._getArtifactPathFromFiles (/Users/paulyu/opensource_project/social-tw-website/node_modules/hardhat/src/internal/artifacts.ts:827:19)
      at Artifacts._getArtifactPath (/Users/paulyu/opensource_project/social-tw-website/node_modules/hardhat/src/internal/artifacts.ts:507:21)
      at async Artifacts.readArtifact (/Users/paulyu/opensource_project/social-tw-website/node_modules/hardhat/src/internal/artifacts.ts:71:26)
      at async getContractFactory (/Users/paulyu/opensource_project/social-tw-website/node_modules/@nomiclabs/hardhat-ethers/src/internal/helpers.ts:99:22)
      at async deployContracts (/Users/paulyu/opensource_project/social-tw-website/packages/relay/test/environment.ts:32:23)
      at async Context.<anonymous> (/Users/paulyu/opensource_project/social-tw-website/packages/relay/test/login.test.ts:19:28)
```

**Additional context**
related to #25 #47

**Comment by NOOMA-42** [comment]
> branch test/login-relay, test/post-relay

branch test/login-relay, test/post-relay, have edited the post

**Comment by ryanycw** [comment]
Solving this in #25

**Comment by ryanycw** [comment]
This is solved in #25
